library/func_misc.py

```diff
@@ -228,7 +234,7 @@ def Call(self, args):
 
         s = val_ops.ToStr(
             val, 'maybe() expected Str, but got %s' % value_str(val.tag()),
-            loc.Missing)
+            args.BlamePos())
```

Comment:
I also wonder if we can have a style like

```
s = val_ops.TryStr(val)
if s is None:
   raise ...
```

Although that doesn't work for bool, int, float, which can't be null